EventFilter/CTPPSRawToDigi/interface/RawDataUnpacker.h

```diff
@@ -26,7 +27,14 @@ class RawDataUnpacker
     typedef uint64_t word;
 
     /// VFAT transmission modes
-    enum { vmCluster = 0x80, vmRaw = 0x90 };
+    enum { vmCluster = 0x80, vmRaw = 0x90, vmDiamondCompact = 0xB0 };
```

Comment:
hi @forthommel - i realize a class with such a generic name as RawDataUnpacker should be in a ctpps namespace. is that a big problem?